src/aave-v2/SupplyVaultBase.sol

```diff
@@ -38,9 +36,7 @@ abstract contract SupplyVaultBase is ERC4626UpgradeableSafe, OwnableUpgradeable
     /// @param _morpho The address of the main Morpho contract.
     constructor(address _morpho) {
         if (_morpho == address(0)) revert ZeroAddress();
-
         morpho = IMorpho(_morpho);
```

Comment:
Careful here because another PR removes the ZeroAddress revert in constructor but here it'd be relevant to keep it

Reply:
Good point! Let's close the other one then